### PR TITLE
add color from 8 to unlimited

### PR DESCRIPTION
**Pull request by zhimin-z**
We can add unlimited human distinguishable colors (though we highly recommend no more than 30 for better distinguishability)

### CODE REVIEW

**Comment by seladb** [comment]
@zhimin-z the `package-lock.json` changes are not needed for this PR, can you remove them?

**Comment by zhimin-z** [comment]
> npm run format:fix

Hi, when I run "npm run format:fix", it changes automatically. I do not to manually change it.

**Comment by seladb** [comment]
> > npm run format:fix
> 
> Hi, when I run "npm run format:fix", it changes package-lock.json automatically. I do not to manually change it. @seladb

I'm not sure why it happens... maybe just not commit this file?

Also please note the build fails:
image

**Comment by zhimin-z** [comment]
Done! @seladb

**Comment by seladb** [comment]
@zhimin-z some of the colors are still too bright... 
Can we keep the existing colors as the first 8 colors and add more after them?

image

**Comment by zhimin-z** [comment]
> @zhimin-z some of the colors are still too bright... Can we keep the existing colors as the first 8 colors and add more after them?
> 
> image

Hi, @seladb I switched to another color picker. Initially, I did add more colors, but it is really hard to manually choose the appropriate color. I've spent quite hours but found myself frustrated. Take a look at this version. It is much better now.

File: src/utils/StargazerLoader.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-  "#F86624",
-  "#00B1F2",
-  "#5A2A27",
-];
-
-export const maxReposAllowed = 8;
+// Refer to post https://stackoverflow.com/questions/10014271/generate-random-color-distinguishable-to-humans
+function makeColor(number) {
+  const hue = number * 137.508; // use golden angle approximation
+  return `hsl(${hue},50%,75%)`;
```
Comment (seladb) [inline src/utils/StargazerLoader.js:7]: Let's change the 3rd param to 50% (`hsl(${hue},50%,50%)`)? The colors look better there:

image

**Review by seladb — APPROVED** [review]

**Pull request merged**